ui/src/shared/components/selectorList/SelectorListEmpty.tsx

```diff
@@ -0,0 +1,25 @@
+// Libraries
+import React, {PureComponent} from 'react'
+
+interface Props {
+  testID: string
+}
+
+export default class BuilderCardEmpty extends PureComponent<Props> {
```

Comment:
Should `BuilderCardEmpty` be `SelectorListEmpty`?

Reply:
I updated it